tests/Unleash.Tests/Utilities/ToggleBootstrapUrlProvider_Tests.cs

```diff
@@ -36,7 +31,8 @@ public void Gets_The_File_Content()
             var responseContent = bootstrapUrlProvider.Read();
 
             // Assert
-            responseContent.Features.Should().BeEmpty();
+            var deserializedResponseContent = JsonSerializer.Deserialize<ToggleCollection>(responseContent);
+            deserializedResponseContent?.Features.Should().BeEmpty();
```

Comment:
And these

Reply:
I assume this is the same as https://github.com/Unleash/unleash-client-dotnet/pull/224#discussion_r1660547726, addressed in the same commit: https://github.com/Unleash/unleash-client-dotnet/pull/224/commits/fe0f1cc65634e24d06fbaaaa1b14cd3e402aeaf8